src/main/resources/templates/users_managment.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<title>User Managment</title>
+<div th:replace="fragmentes/header :: header-css" />
+</head>
+<body>
+<div th:replace="fragmentes/header :: header" />
+<div class="container my-2 "  style="text-align: center">
+    <h1  th:text="${showText}"></h1>
+<div style="text-align: left ; margin: 5px " >
+    <a th:href="@{/showNewUserForm}" class="btn btn-primary btn-sm mb-3"  >Add New User </a>
+
+</div>
+    <table  class ="table table-striped table-responsive-sm table-bordered" >
+        <thead class="thead-dark" style="background-color: cadetblue" >
+        <tr >
+            <th style="text-align: center"> Userid </th>
```

Comment:
Use string resource file for constant strings and then remove these constant strings.